dart/lib/src/hub.dart

```diff
@@ -176,6 +178,13 @@ class Hub {
     return sentryId;
   }
 
+  Future<void> captureUserFeedback(UserFeedback userFeedback) async {
+    final item = _peek();
+    await item.client.captureEnvelope(
```

Comment:
`client` could expose `captureUserFeedback` that does the `SentryEnvelope.fromUserFeedback` internally